package-system/sqlite/build_config.json

```diff
@@ -0,0 +1,46 @@
+{
+   "git_url":"https://github.com/sqlite/sqlite.git",
+   "git_tag":"version-3.32.2",
+   "package_name":"SQLite",
+   "package_version":"3.32.2-rev1",
+   "package_url":"https://www.sqlite.org",
+   "package_license":"CUSTOM",
+   "package_license_file":"copyright.txt",
+   "cmake_find_source":"FindSQLite.cmake",
+   "cmake_find_target":"FindSQLite.cmake",
+   "additional_src_files":[
+      "copyright.txt"
+   ],
+   "Platforms":{
+      "Windows":{
+         "Windows":{
+            "custom_build_cmd": [
+               "build_windows.cmd"
+            ],
+            "custom_install_cmd": [
+               "package_windows.cmd"
+            ]
+         }
+      },
+      "Darwin":{
+         "Mac":{
+            "custom_build_cmd": [
+               "./build_linux.sh"
+            ],
+            "custom_install_cmd": [
+               "./package_linux.sh"
```

Comment:
Seems like `build_unix.sh` and `package_unix.sh` is a more accurate name for these scripts.

Reply:
```build_unix_like.sh``` and ```package_unix_like.sh``` ? (from https://sqlite.org/src/doc/trunk/README.md )

Reply:
"unix_like" is more accurate and I am good with that suggestion.